pkgs/development/compilers/dotnet/default.nix

```diff
@@ -51,6 +51,13 @@ makeScopeWithSplicing' {
       patchNupkgs = callPackage ./patch-nupkgs.nix {};
       nugetPackageHook = callPackage ./nuget-package-hook.nix {};
 
+      buildDotnetModule = callPackage ../../../build-support/dotnet/build-dotnet-module { };
+      buildDotnetGlobalTool = callPackage ../../../build-support/dotnet/build-dotnet-global-tool { };
+
+      mkNugetSource = callPackage ../../../build-support/dotnet/make-nuget-source { };
+      mkNugetDeps = callPackage ../../../build-support/dotnet/make-nuget-deps { };
+      fetchNupkg = callPackage ../../../build-support/dotnet/fetch-nupkg { };
```

Comment:
We can leave this for later, but it'd be nice to document these so people know what they are and what they do

Reply:
I don't think `mkNugetSource/Deps` serve a purpose now, and hopefully `fetchNupkg` can replace `fetchNuGet`.